src/script/view_model/content/GroupCreationViewModel.ts

```diff
@@ -175,10 +175,10 @@ export class GroupCreationViewModel {
     const {value} = event.target;
 
     const trimmedNameInput = value.trim();
-    const nameTooLong = trimmedNameInput.length > this.maxNameLength;
+    const nameTooLong = value.length > this.maxNameLength;
     const nameTooShort = !trimmedNameInput.length;
 
-    this.groupName(trimmedNameInput);
```

Comment:
Actually the fix for this might only be to not update the observable with the trimmed value but rather with the actual value. (eg. deleting this line of code). 

We would still need to trim once the user validates the creation (maybe it's already done)

I am not entirely sure what would be the side effects of not updating the observable at this point, but it's worth trying

Reply:
If i remove this line, the group name doesn't update. In my change i used value which user type.

Sure, the trim should be prepared on creation :)

Reply:
Yes, that makes sense. We might need to change a tiny bit how the component works.